CHANGELOG.md

```diff
@@ -222,7 +228,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Ansible playbook for applying csm packages to Compute and Application nodes
 
-[Unreleased]: https://github.com/Cray-HPE/csm-config/compare/1.15.21...HEAD
+[Unreleased]: https://github.com/Cray-HPE/csm-config/compare/1.15.24...HEAD
+
+[1.15.24]: https://github.com/Cray-HPE/csm-config/compare/1.15.23...1.15.24
+
+[1.15.23]: https://github.com/Cray-HPE/csm-config/compare/1.15.22...1.15.23
+
+[1.15.22]: https://github.com/Cray-HPE/csm-config/compare/1.15.21...1.15.22
```

Comment:
Not exactly sure why this changed in my PR.

Reply:
I added it -- the TOC at the bottom of the changelog hadn't been updated (by you, or the past couple PRs, looked like)

Reply:
In fact, diffing this to `release/1.15` doesn't show this in my local branch.
```
(Shasta) [671]rusty@HPE-XHD22YD7DW:~/gitstuffs/cray-shasta/csm-config> git diff release/1.15 CHANGELOG.md
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 3e46bb2..266629f 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.15.24] - 2024-01-24
+
+- [MTL-2348](https://jira-pro.it.hpe.com:8443/browse/MTL-2348: New `ncn_kernel_upgrade` role. This role is now included in the `ncn_initrd.yml` playbook. These changes will
+  enforce the installation of the new SuSE PTF Kernel and Marvell KMP. Module tweaks are also included, preventing `qedr`
+  from loading during the rootfs, as well as being excluded from dracut.
+
 ## [1.15.23] - 2023-10-13
 
 ### Changed

```

Reply:
That should be updated every time a new version entry is added

Reply:
I added it via the github web interface -- it didn't come from your end.